nox/sessions.py

```diff
@@ -115,6 +116,11 @@ def bin(self):
         """The bin directory for the virtualenv."""
         return self._runner.venv.bin
 
+    @property
+    def interactive(self):
+        """Boolean showing if the session was ran in an interactive sessions."""
```

Comment:
Use active voice, please: `Returns True if the Nox is being run in an interactive session or False otherwise.`

Reply:
Just changed it in the latest commit